src/screens/SignTransactionSheet.tsx

```diff
@@ -507,15 +498,15 @@ export const SignTransactionSheet = () => {
           fn: signPersonalMessage,
           screen: SCREEN_FOR_REQUEST_SOURCE[source],
           operation: TimeToSignOperation.SignTransaction,
-        })(message, existingWallet);
+        })(message, provider, existingWallet);
```

Comment:
had to switch the order of the arguments for both `signPersonalMessage` and `signTypedDataMessage` as they no longer have an optional provider but still can have an optional wallet arg